docs/running-tests-linux-locally.md

```diff
@@ -0,0 +1,39 @@
+# Running Tests on Linux Locally
+
+There are currently two ways to run tests on Linux locally:
+
+- Using WSL (Windows Subsystem for Linux)
+- Using Docker
+
+## Running Tests with WSL
+
+### Prerequisites
```

Comment:
please also mention that it's prerequisite to run the script for building the scanner first.